js/rtorrent.js

```diff
@@ -666,19 +668,27 @@ rTorrentStub.prototype.makeMultiCall = function()
 	}
 	else
 	{
+		// fragmentation of xml command (Content-Length must be <2MB for rtorrent 0.9.8)
+		var maxContentSize = 2 << (20 + 3*(theWebUI.systemInfo.rTorrent.apiVersion>=11));
 		this.content+='system.multicall</methodName><params><param><value><array><data>';
-		for(var i=0; i<this.commands.length; i++)
+		this.hashes = [];
+		for(; this.commandOffset < this.commands.length; this.commandOffset++)
 		{
-			var cmd = this.commands[i];
-			this.content+=('<value><struct><member><name>methodName</name><value><string>'+
+			var cmd = this.commands[this.commandOffset];
+			var cmd_string=('<value><struct><member><name>methodName</name><value><string>'+
 				cmd.command+'</string></value></member><member><name>params</name><value><array><data>');
 			for(var j=0; j<cmd.params.length; j++)
 			{
 				var prm = cmd.params[j];
-				this.content += ('<value><'+prm.type+'>'+
+				cmd_string += ('<value><'+prm.type+'>'+
 					prm.value+'</'+prm.type+'></value>');
 			}
-			this.content+="</data></array></value></member></struct></value>";
+			cmd_string+="</data></array></value></member></struct></value>";
+			if (this.hashes.length > 0 && this.content.length + cmd_string.length + 31 + 22 > maxContentSize)
```

Comment:
As we may see above, initially ```this.hashes = [];``` i.e. ```this.hashes.length == 0;```.
I.e. at the first iteration this condition always false and this cycle will run only once.

Reply:
Same thing here: at least one command should be included in a request